nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSDocument.java

```diff
@@ -198,6 +198,13 @@
 
     public static final String APPLICATION_OCTET_STREAM = "application/octet-stream";
 
+    public static final String UID_MAJOR_VERSION = "uid:major_version";
+
+    public static final String UID_MINOR_VERSION = "uid:minor_version";
+
+    public static final String SUFFIX_MAJOR_VERSION = "major_version";
```

Comment:
No need for `SUFFIX`. But please call these 4 `PROP_something` to stress the fact that they are "properties" names from the high level representation of documents, not low-level "keys".

So use `PROP_UID_MAJOR_VERSION`, `PROP_UID_MINOR_VERSION`, `PROP_MAJOR_VERSION`, `PROP_MINOR_VERSION`.

Reply:
It would be great if there were some constants I could reuse.